### PR TITLE
suggesting few changes in article text

### PR DESCRIPTION
**Pull request by bappyasif**
#### Summary

found some potentially adjustable changes in `other form control` article

i considered, sourcing text suitable for suggested changes from other mdn article, to make it more ease for readers
#### Motivation

while i was reading through this article some fixes seem to me quite obvious, so i took this opportunity to share it with you all, 

and please feel free to correct me if i should have addressed it differently 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/forms/other_form_controls/index.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ For example, in browsers that support `{{htmlelement("datalist")}}` on `range` i
 
 And browsers that support {{htmlelement('datalist')}}s and [`<input type="color">`](/en-US/docs/Web/HTML/Element/input/color) should display a customized palette of colors as the default, while still making the full color palette available.
 
-In this case, different browsers behave differently from case to case, so consider such uses as progressive enhancement, and ensure they degrade gracefully.
+In this case, different browsers behave differently from case to case, so consider such uses as progressive enhancement, and ensure they fallback gracefully.
```
Comment (OnkarRuikar) [inline files/en-us/learn/forms/other_form_controls/index.md:246]: We can break the long sentence:
```suggestion
In this case, different browsers behave differently from case to case. So, consider such uses as progressive enhancement and ensure they fallback gracefully.
```

File: files/en-us/learn/forms/other_form_controls/index.md [hunk 2]
```diff
@@ -267,7 +267,7 @@ A progress bar represents a value that changes over time up to a maximum value s
 
 This is for implementing anything requiring progress reporting, such as the percentage of total files downloaded, or the number of questions filled in on a questionnaire.
 
-The content inside the {{HTMLElement("progress")}} element is a fallback for browsers that don't support the element and for screen readers to vocalize it.
+The content inside the {{HTMLElement("progress")}} element is a fallback for browsers that don't support this element and for screen readers to vocalize it.
```
Comment (OnkarRuikar) [inline files/en-us/learn/forms/other_form_controls/index.md:270]: What's wrong with `the`? There are no other elements involved in the discussion.

Comment (bappyasif) [reply files/en-us/learn/forms/other_form_controls/index.md:270]: @OnkarRuikar nothing wrong but reads well with 'this', and becomes more specific, as to 'the' element, won't you agree, thanks :)

**Comment by wbamberg** [comment]
As with the other similar PRs, (https://github.com/mdn/content/pull/15008, https://github.com/mdn/content/pull/15006), I'm closing this one as it's not clear the changes are really an improvement over the original.

**Pull request closed without merging**